.github/in-cluster-test-scripts/eks-tunnel.sh

```diff
@@ -3,6 +3,10 @@
 set -x
 set -e
 
+# Create a FIFO through which we'll pipe the sysdump before the job terminates.
+# NOTE: This MUST be the first step in the script.
```

Comment:
nitpick - why does it have to be the first step in the script?

Reply:
That is to reduce the failure window for `kubectl exec -- cat /fifo` when `/fifo` does not exist — for that reason, I guess we should always create them as early as possible.